internal/resource/store/fileresourcestore_test.go

```diff
@@ -37,8 +39,8 @@ func (s *fileResourceStoreSuite) SetUpTest(c *gc.C) {
 	fp := fmt.Sprintf("%x", sha384hash.Sum(nil))
 	fingerprint, err := charmresource.ParseFingerprint(fp)
 	c.Assert(err, jc.ErrorIsNil)
-	s.resource = Resource{
-		UUID: resourcestesting.GenResourceUUID(c),
+	s.resource = coreresource.Resource{
+		ID: resourcestesting.GenResourceUUID(c).String(),
```

Comment:
**question:** Why move back to `ID` ? We use to try to name things by their name, so we should've keep `UUID` IMHO.

Reply:
We're using a different structure now which has the value as ID: https://github.com/juju/juju/blob/main/core/resource/resource.go#L42-L46 
Where it actually is an ID. We'll need to resolve that in the future.